### PR TITLE
Copy Edits to Readme.md

### PR DESCRIPTION
**Pull request by aaron-smartthings**
Copy edits to the readme file for usability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,46 @@
...
+2. [Yarn](https://yarnpkg.com) or [NPM](https://www.npmjs.com/)
 
-To deploy the _Rule Helper_, make sure you have the latest version of [NodeJS](https://nodejs.org)
-installed at your computer.
+#### Installation
+This tutorial uses Yarn as package manager. However, you can still use NPM if you prefer. Navigate to your Rule Helper and run the following command to install the app dependencies.
 
-Additionally, this deployment tutorial uses [Yarn](https://yarnpkg.com) as package manager, however,
-you're not limited to it. You can still use [NPM](https://www.npmjs.com/) if you prefer.
+    `cd Rule-Helper
```
Comment (erickvneri) [inline README.md:35]: Repo will be downloaded lowercased, therefore this should be _rule-helper/_.

**Review by aaron-smartthings — COMMENTED** [review]
fixed capitalization

**Review by aaron-smartthings — COMMENTED** [review]
copy edit capitalization

**Comment by erickvneri** [comment]
Great!

**Pull request merged**